packages/verified-fetch/src/verified-fetch.ts

```diff
@@ -430,11 +442,35 @@ export class VerifiedFetch {
     [identity.code]: this.handleRaw
   }
 
+  /**
+   *
+   * TODO: Should we use 400, 408, 418, or 425, or throw and not even return a response?
+   */
+  private async abortHandler (opController: AbortController): Promise<void> {
+    this.log.error('signal aborted by user')
+    opController.abort('signal aborted by user')
+    await this.stop()
```

Comment:
we should discuss this as well. without calling stop, sometimes things don't quit. bug in Helia? should we call `.start()` immediately after?

Reply:
removed calling of `this.stop()`